xds/internal/xdsclient/v2/rds_test.go

```diff
@@ -111,11 +111,11 @@ func (s) TestRDSHandleResponseWithRouting(t *testing.T) {
 					VirtualHosts: []*xdsclient.VirtualHost{
 						{
 							Domains: []string{uninterestingDomain},
-							Routes:  []*xdsclient.Route{{Prefix: newStringP(""), WeightedClusters: map[string]xdsclient.WeightedCluster{uninterestingClusterName: {Weight: 1}}}},
+							Routes:  []*xdsclient.Route{{Prefix: newStringP(""), WeightedClusters: map[string]xdsclient.WeightedCluster{uninterestingClusterName: {Weight: 1}}, RouteAction: xdsclient.RouteActionRoute}},
```

Comment:
Optional: The `Route` now has enough fields set that it can be broken into multiple lines. This and other places in this file.

Reply:
Done.